content/blog/color-schemes.md

```
@@ -0,0 +1,209 @@
+
+```toml
+[params.colorSchemeSwitcher.selector.disable]
+header = false
+footer = true   # hide in footer, show only in header
+```
+
+> **Useful during development too.** Even if you don't ship the switcher to visitors, enabling it while you work lets you preview every scheme against your actual content — profile photo, project thumbnails, blog posts — without editing `hugo.toml` and restarting the server each time. Once you've settled on a color, set `colorScheme` and disable the switcher.
+
+You can also embed the picker inline on any page or post with a shortcode:
+
+```
```

markdownlint-cli2 (0.23.0)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)